app/src/main/java/com/example/androidpractice/domain/model/Rating.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.androidpractice.domain.model
+
+class Rating(
+    val kinopoisk: Double,
+    val imdb: Double,
+    val rottenTomatoes: Double,
+) {
+}
```

Comment:
может сделать дата классом?

```suggestion
data class Rating(
    val kinopoisk: Double,
    val imdb: Double,
    val rottenTomatoes: Double,
)
```

Reply:
done